src/dyz.c

```diff
@@ -10,6 +12,11 @@ int main(int _argc, char** _argv)
     argc = _argc;
     argv = _argv;
 
+#ifdef LUASEARCHPATH
```

Comment:
This is not needed if the Lua sources get linked into dyz from .o files containing bytecode.